skyportal/handlers/api/observation_plan.py

```diff
@@ -575,6 +579,54 @@ def get(self, observation_plan_request_id=None):
           tags:
             - observation_plan_requests
           parameters:
+            - in: query
+              name: dateobs
+              nullable: true
+              schema:
+                type: string
+              description: GcnEvent dateobs to filter on
+            - in: query
+              name: instrumentID
+              nullable: true
+              schema:
+                type: integer
+              description: Instrument ID to filter on
+            - in: query
+              name: startDate
+              nullable: true
+              schema:
+                type: string
+              description: |
+                Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
+                created_at >= startDate
+            - in: query
+              name: endDate
+              nullable: true
+              schema:
+                type: string
+              description: |
+                Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
+                created_at <= endDate
+            - in: query
+              name: status
+              nullable: true
+              schema:
+                type: string
+              description: |
+                String to match status of request against
+            - in: query
+              name: numPerPage
+              nullable: true
+              schema:
+                type: integer
+              description: |
+                Number of observation plan requests to return per paginated request. Defaults to 100. Can be no larger than {MAX_FOLLOWUP_REQUESTS}.
```

Comment:
I'm not sure this is an f-string. 
I'm also surprised you did not notice that, since modern IDEs will highlight a value in an f-string and not highlight curved brackets in a regular string...

Reply:
The salt is strong with this one.